### ISSUE TITLE
Accessibility: `ALT+0` help menu doesn't work

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-admin#2003]
According to https://www.tiny.cloud/docs/tinymce/6/tinymce-and-screenreaders/ and read out to (some) screen readers, pressing `ALT+0` should display a help menu. This works as expected in [the examples in tinymce website](https://www.tiny.cloud/docs/tinymce/6/basic-example/).

Without this, keyboard users have no way of knowing how to access the menu items.

## Acceptance Criteria

- [ ] When focus is inside the TinyMCE editor, pressing `ALT+0` brings up the accessibility menu
- [ ] There is some visual indicator for sighted keyboard users that pressing `ALT+0` will open this menu
